about/index.html

```diff
@@ -241,19 +266,41 @@ <h3>Python Support</h3>
             of volumes.</p>
           </div>
 
+          <div class="feature">
+            <h3>Point Clouds and Tools</h3>
+            <p>OpenVDB can natively store point data within its hierarchy using PointDataGrids.
+            These grids can store the points with attributes directly in the VDB Grid. Points are
+            spatially-organized into VDB voxels to provide faster access and a greater opportunity
+            for data compression compared with linear point arrays.
+            </p>
+          </div>
+
+          <div class="feature-right">
+            <h3>AX: Domain Specific Language</h3>
+            <p>OpenVDB AX is a DSL which can be built alongside OpenVDB. It is an expression
+            language where expressions are JIT-compiled and used with a tailored multi-threaded
+            execution framework, specifically designed for VDB. AX promises high performance
+            and custom manipulation for VDB based kernels.
+            </p>
+          </div>
+
```

Comment:
we should ad a similar paragraph for NanoVDB

Reply:
Would you mind writing one I can put here?

Reply:
Done